StreetCare/StreetCare/Views/Screens/OutreachFormView.swift

```diff
@@ -254,7 +254,195 @@ struct OutreachFormView: View {
+
+                DatePicker(NSLocalizedString("endDate", comment: ""), selection: $endDate, displayedComponents: .date)
+                    .datePickerStyle(CompactDatePickerStyle())
+
+                DatePicker(NSLocalizedString("endTime", comment: ""), selection: $endTime, displayedComponents: .hourAndMinute)
+                    .datePickerStyle(CompactDatePickerStyle())
+
+                // Help Type
+                Text(NSLocalizedString("helpType", comment: ""))
+                    .font(.headline)
+
+                TextField(NSLocalizedString("eghelpType", comment: ""), text: $helpType)
+                    .textFieldStyle(RoundedBorderTextFieldStyle())
+
+                // Maximum Capacity
+                Text(NSLocalizedString("maximumCapacity", comment: ""))
+                    .font(.headline)
+
+                TextField(NSLocalizedString("egmaximumCapacity", comment: ""), text: $maxCapacity)
+                    .textFieldStyle(RoundedBorderTextFieldStyle())
+                    .keyboardType(.numberPad)
+
+                // Event Description
+                Text(NSLocalizedString("eventDescription", comment: ""))
+                    .font(.headline)
+
+                TextEditor(text: $eventDescription)
+                    .frame(height: 200)
+                    .border(Color.gray, width: 0.5)
+                    .cornerRadius(5)
+                    .onChange(of: eventDescription) { newValue in
+                        if newValue.count > descriptionLimit {
+                            eventDescription = String(newValue.prefix(descriptionLimit))
+                        }
+                    }
+
+                HStack {
+                    Spacer()
+                    Text("\(eventDescription.count)/\(descriptionLimit)")
+                        .font(.caption)
+                        .foregroundColor(.gray)
+                }
+
+                // Buttons Section
+                VStack(spacing: 20) {
+                    NavLinkButton(title: NSLocalizedString("selectSkillsButtonTitle", comment: ""), width: 300, secondaryButton: false)
+                        .onTapGesture {
+                            isSkillSheetPresented.toggle()
+                        }
+
+                    NavLinkButton(title: NSLocalizedString("saveEventButtonTitle", comment: ""), width: 300, secondaryButton: false)
+                        .disabled(!allFieldsFilled)
+                        .onTapGesture {
+                            saveToFirestore()
+                        }
+
+                    NavLinkButton(title: NSLocalizedString("discardButtonTitle", comment: ""), width: 300, secondaryButton: true)
+                        .onTapGesture {
+                            isPresented = false // Close both views on discard
+                        }
+                }
+                .frame(maxWidth: .infinity, alignment: .center)
+                .padding()
+            }
+            .padding()
+        }
```

Comment:
Is the code for the select skills sheet and overlay missing? Please review the previous working code and ensure it's still carried over into your new changes